docs/windows/policy/en-US/tailscale.adml

```diff
@@ -0,0 +1,206 @@
+
+If you disable this policy, the Tailscale standard log server will be used by default, but a non-standard Tailscale log server can be configured using the TS_LOG_TARGET environment variable.]]></string>
+            <string id="Tailnet">Login Tailnet</string>
+            <string id="Tailnet_Help"><![CDATA[This policy can be used to suggest or require a specific tailnet when opening the login page.
+See https://tailscale.com/kb/1315/mdm-keys#set-a-suggested-or-required-tailnet for more details.
+
+To suggest a tailnet at login time, set this to the name of the tailnet, as shown in the top-left of the admin panel, such as "example.com". That tailnet's SSO button will be shown prominently, along with the option to select a different tailnet.
+
+To require logging in to a particular tailnet, add the "required:" prefix, such as "required:example.com". The result is similar to the suggested tailnet but there will be no option to choose a different tailnet.
+
+If you configure this policy, set it to the name of the tailnet, possibly with the "required:" prefix, as described above.
+
+If you disable this policy, the standard login page will be used.]]></string>
+            <string id="ExitNodeID">Require Exit Node</string>
+            <string id="ExitNodeID_Help"><![CDATA[This policy can be require always using the specified Exit Node whenever the Tailscale client is connected.
+See https://tailscale.com/kb/1315/mdm-keys#force-an-exit-node-to-always-be-used and https://tailscale.com/kb/1103/exit-nodes for more details.
+
+If you enable this policy, set it to the ID of an exit node. The ID is visible on the Machines page of the admin console, or can be queried using the Tailscale API. If the specified exit node is unavailable, this device will have no Internet access unless Tailscale is disconnected.
+
+If you disable this policy or supply an empty exit node ID, then usage of exit nodes will be disallowed.
+
+If you do not configure this policy, no exit node will be used by default but an exit node (if one is available and permitted by ACLs) can be chosen by the user if desired.]]></string>
+            <string id="AllowIncomingConnections">Allow Incoming Connections</string>
+            <string id="AllowIncomingConnections_Help"><![CDATA[This policy can be used to require that the Allow Incoming Connections setting is configured a certain way.
+See https://tailscale.com/kb/1315/mdm-keys#set-whether-to-allow-incoming-connections and https://tailscale.com/kb/1072/client-preferences#allow-incoming-connections for more details.
+
+If you enable this policy, then Allow Incoming Connections is always enabled and the menu option is hidden.
+If you disable this policy, then Allow Incoming Connections is always disabled and the menu option is hidden.
+If you do not configure this policy, then Allow Incoming Connections depends on what is selected in the Preferences submenu.]]></string>
+            <string id="UnattendedMode">Enable Run Unattended</string>
```

Comment:
nit: As a general rule, we should use more descriptive policy names as the help text might be unavailable in certain contexts. An example would be a group policy report that looks like this:
![image](https://github.com/tailscale/tailscale/assets/1761190/a49e820d-e3db-4fb8-9500-1e1482573868)

To make policy names less ambiguous and emphasize that these policies actually change the Tailscale client behavior rather than just enable certain option or capability, we might want to rename them to something like this:
- Run Tailscale in Unattended Mode
- Run Tailscale as an Exit Node

Similarly, we might want to rename "Require Exit Node" to "Use the specified Exit Node" or something similar to emphasize not only that an exit node is required but also that the policy configures which exit node to use, etc.

Reply:
I only changed the items that didn't have a clear directive. Does this look better?